aten/src/ATen/native/eigen/SparseBlas.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <ATen/Tensor.h>
+
+namespace at::native::eigen::sparse {
```

Comment:
I think the namespace here should be `at::native::sparse::impl::eigen` to follow pattern  indicating the functions are operating at the same level as those provided in `at::native::sparse::impl::mkl`.